Makefile

```diff
@@ -64,6 +64,30 @@ install:
 		helm upgrade $(HELM_OPTS) $(CHART_INSTANCE) $(CHART_DIR); \
 	fi;
 
+.PHONY: install_giraffe
+install_giraffe: ## Installs the helm chart on the OpenShift cluster
+install_giraffe: check_environment
+install_giraffe:
+install_giraffe: GIT_SHA1=$(shell git rev-parse HEAD)
+install_giraffe: IMAGE_TAG=$(GIT_SHA1)
+install_giraffe: NAMESPACE=$(CIF_NAMESPACE_PREFIX)-tools
```

Comment:
Are we planning to continue deploying this to -tools for now?

Reply:
I'd be tempted to? Unless we're fine having everything in the same place, and we have the resources to do that